.travis.yml

```diff
@@ -7,7 +7,7 @@ node_js:
   - "8"
 
 addons:
-  firefox: "56.0"
+  firefox: "62.0"
```

Comment:
Does this mean it’s a breaking change, and we’ll need to update our Firefox version?

Reply:
It shouldn't be based on the changes I made. What version are you on currently? It's possible that all we need is a smaller version bump.

Reply:
At first I thought it might be an update to selenium-webdriver that caused this, but we specify `^3.6.0` and the latest version matching that range is `3.6.0`.

Reply:
Ah, but that leaves `geckodriver`. It has a few versions published after `1.9.0` (which is what we depend on, `^1.9.0`). Let me check if locking that version down affects anything.

Reply:
we’re on 56; we had to update from 52 last time.

Reply:
ah, yeah let's see if we can make things work on 56 then.

Reply:
Okay, locking down geckodriver seems to work. No update required :).